lib/logstash/inputs/redis.rb

```diff
@@ -77,6 +94,17 @@ def new_redis_instance
     @redis_builder.call
   end
 
+  def init_threadpool
+    @threadpool_queue_max ||= 2 * @pattern_list_threads
```

Comment:
This probably doesn't need to be stored instance variable since I can't find any usage outside this method? If so, we can probably skip the variable assignment altogether and use it directly

Reply:
Yeah, right, this was used elsewhere at some point